package.json

```diff
@@ -17,7 +17,33 @@
         	"command": "extension.loadLsifFile",
         	"title": "Load LSIF File"
       	  }
-    	]
+    	],
+    	"configuration": {
+        	"title": "Unicon LSP Main",
+        	"properties": {
+            	"lspMain.enableDebugLogs": {
+                	"type": "boolean",
+                	"default": false,
+                	"description": "Enable LSIF debug logging."
+            	},
+            	"lspMain.mode": {
+                	"type": "string",
+                	"default": "Both",
+                	"enum": ["LSIF only", "LSP only", "Neither", "Both"],
+                	"description": "Choose how to handle language support: LSIF only, LSP only, Neither, or Both."
```

Comment:
There is a mix of tab an space characters in this section and  other places down in the PR. Can you please fix those? Be consistent with the existing white space.